Formula/cartesi-machine-linux-image.rb

```diff
@@ -0,0 +1,18 @@
+class CartesiMachineLinuxImage < Formula
+  desc "Kernel image for the Cartesi Machine"
+  homepage "https://cartesi.io/"
+  VERSION = "0.20.0".freeze
+  LINUX_IMAGE = "linux-6.5.13-ctsi-1-v#{VERSION}.bin".freeze
+  url "https://github.com/cartesi/image-kernel/releases/download/v#{VERSION}/#{LINUX_IMAGE}"
+  sha256 "65dd100ff6204346ac2f50f772721358b5c1451450ceb39a154542ee27b4c947"
+  license "Apache-2.0"
```

Comment:
```suggestion
  license "GPL-2.0"
```

Reply:
But Linux is GPL-2.0 , also Debian/ArchLinux distributes its Linux images as GPL-2.0. Our `machine-linux-repository` is Apache-2.0 but it doesn't make the compiled image Apache-2.0 I think. So the build script is Apache-2.0, but the generated artifact is GPL-2.0, kinda confusing, but is this, right? Or am I wrong?